tests/WorkerTests.txt

```diff
@@ -0,0 +1,7 @@
+testTitle=UnitTests
```

Comment:
I think in 7.0+ we use toml files for tests?

Reply:
I think in release branch, it is toml files. In master branch, it is .txt files.

Reply:
No no, at least in 6.3, we are using txt for sure. Can you double check?

Reply:
It seems that all the unittest located here https://github.com/apple/foundationdb/tree/master/tests uses .txt